### DISCUSSION TITLE
Browser console showing errors

### DISCUSSION THREAD

**Discussion by ashishslt** [livewire/livewire#8715] [1. Help]
I build a site using Laravel and Livewire. when the page loads there are no errors on browser console, but after 1-2 mins some uncaught syntax error comes which says this function has already been declared. (ex: Uncaught SyntaxError: Identifier 'AdSizes' has already been declared ). Please check attachment as well.

![Screenshot 2024-08-01 171246](https://github.com/user-attachments/assets/b1067bf7-6dbf-45ee-b606-d0ec01ce61c0)

**Comment by vikasdeepslt** [comment]
facing similar issue with my application. @ashishslt @andreascreten
